examples/platform/linux/AppMain.cpp

```diff
@@ -0,0 +1,144 @@
+
+
+#include <iostream>
+#include <thread>
+
+#include <platform/CHIPDeviceLayer.h>
+#include <platform/PlatformManager.h>
+
+#include <app/server/Server.h>
+#include <core/CHIPError.h>
+#include <setup_payload/QRCodeSetupPayloadGenerator.h>
+#include <setup_payload/SetupPayload.h>
+#include <support/CHIPMem.h>
+#include <support/RandUtils.h>
+
+#include "Options.h"
+
+using namespace chip;
+using namespace chip::Inet;
+using namespace chip::Transport;
+using namespace chip::DeviceLayer;
+
+namespace {
+void EventHandler(const chip::DeviceLayer::ChipDeviceEvent * event, intptr_t arg)
+{
+    (void) arg;
+    if (event->Type == chip::DeviceLayer::DeviceEventType::kCHIPoBLEConnectionEstablished)
+    {
+        ChipLogProgress(DeviceLayer, "Receive kCHIPoBLEConnectionEstablished");
+    }
+}
+
+CHIP_ERROR PrintQRCodeContent()
```

Comment:
Do you think it can be replaced with `ConfigurationMgr().LogDeviceConfig()` + `PrintOnboardingCodes` like the other examples or you need to print it on stderr?

Reply:
Yes, these code is written before these two functions are added. Removed them and use LogDeviceConfig + PrintOnboardingCodes